config/initializers/flipper.rb

```diff
@@ -0,0 +1,6 @@
+class CanAccessFlipperUI
```

Comment:
I think this code should live in `lib/constraints/` or similar (it's said here https://api.rubyonrails.org/classes/ActionDispatch/Routing/Mapper/Scoping.html)

The problem is that we'll need to add `lib/constraints` to the autoload paths. Maybe it's just easier to move it to `app/constraints` ?

I don't like an initializer because they have their own lifecycle (they are kinda special), and I don't see the need to have a plain Ruby class defined there.